pkgs/tools/security/mitm6/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, python3
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "mitm6";
+  version = "0.3.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-g+eFcJdgP7CQ6ntN17guJa4LdkGIb91mr/NKRPIukP8=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    scapy
+    future
+    twisted
+    netifaces
+  ];
+
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "mitm6"
+  ];
+
+  meta = with lib; {
```

Do not use nested `lib`.

```suggestion
  meta = {
```